3001-4000/LIT3592Miracle.xml

```diff
@@ -5,18 +5,21 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       <fileDesc>
          <titleStmt>
             <title xml:id="t1" xml:lang="en">Miracle of Mary: Miracle of the monk Isaac</title>
-            <title xml:id="t2" xml:lang="en">Miracle of the monk Yǝsḥāq</title>
-            <title xml:id="t3" xml:lang="en">The monk, Yǝsḥaq, who prayed for seven years that St Mary would appear to him</title>
+            <title xml:id="t2" xml:lang="en">Miracle of Mary: Miracle of the priest Isaac</title>
+            <title xml:id="t3" xml:lang="en">The monk, Yǝsḥaq, who prayed for seven years that St
```

Comment:
I assume that t3 and t4 have nothing to do with this GENERAL RECORD at all, as it is a generic ID to be used for ANY miracle of Yeshaq when one does not know which one exactly